articles/fuzzy-logic-operations/index.md

```diff
@@ -0,0 +1,138 @@
+ 
+The above equation can be viewed as a collection of ordered pairs. The inclusion of the element $S \epsilon\; x$ into the fuzzy set $F$ is fuzzy.
+
+#### Example
+
+Consider the example of temperature control. We will explore the various membership functions using this example. The task to control the room temperature can be modeled as a fuzzy problem. Let the ranges of temperature be given as follows:
+
+$$
+70 \le Temperature \le 80 \;: Low \\
+80 \le Temperature \le 90 \;: Medium \\
+90 \le Temperature \le 100 \;: High\\
+$$
+
+Graphically the problem can be represented as the following:
+
+![graphical representation](graphical_rep_1.png)
+
+The same problem can be represented using fuzzy. Observe how a given temperature belongs to two classes of temperature in varying degrees. 
+
+![graphical representation](graphical_rep_2.png)
+
+### Types of membership functions
+
+As seen in the earlier example, we used a triangular membership function to represent the temperatures. However, the triangular membership function is not suited for all applications. Therefore, many other membership functions were introduced to widen the scope of applications.
+
+#### Gamma ( $\gamma$ ) function
+
+Mathematically, the gamma function is given as follows:
+
+$$ \gamma(x,\alpha, \beta) = \begin{cases} 0 & x\le \alpha \\
+\frac{x-\alpha}{\beta -\alpha} & \alpha\le\beta \\ 1 & x \ge \beta\end{cases}  $$
+
+![gamma](gamma-correct.png)
+#### s- function
+
+The s function gets its name from its shape. The s-shaped function is useful in cases with a gradual change in the membership. 
+
+Mathematially, it is given as:
+
+$$ S(x,\alpha,\beta) = \begin{cases}
+0 & x\le \alpha \\
+2*(\frac{(x -\alpha)}{\gamma - \alpha})^2 &\alpha \lt x \le \beta \\
+1 - 2*(\frac{(x -\alpha)}{\gamma - \alpha})^2 & \beta \lt x \le \gamma \\
+1 & x \gt \gamma
+\end{cases} $$
+
+![s-function](gamma.png)
+
+#### L- function
+This function is the inverse of $\gamma$ function. It is given as follows:
+
+$$ L(x,\alpha,\beta) = \begin{cases} 
+1 & x \le \alpha \\
+\frac{\alpha - x}{\beta - x} & \alpha \le x \le \beta \\ 
+0 & x \gt \beta 
+\end{cases} $$
+ 
+![L-function](L-function.png)
+#### Triangular function
+
+$$ L(x,\alpha,\beta,\gamma) = \begin{cases} 
+0 & x\le \alpha \\
+\frac{x-\alpha}{\beta - \alpha} & \alpha \le x \le \beta \ \\
+\frac{\alpha - x}{\beta - \alpha} & \beta \le x \le \gamma \\
+0 & x \gt \gamma
+\end{cases}$$
+
+![triangular function](triangular_function.png)
+
+#### pi function
+
+This function gets its name from the shape of pi. This function looks similar to the symbol pi ($\pi$). Mathematically it is given as follows:
+
+$$ \pi(x, \alpha, \beta,\gamma, \delta) = \begin{cases} 
+0 & x\le \alpha \\
+\frac{x - \alpha }{\beta - \alpha} & \alpha \lt x \le \beta \ \\
+1 & \beta \lt x \le \gamma \\
+\frac{\gamma - x }{\delta - \gamma} & \gamma \lt x \le \delta \
+\end{cases}$$
+
+![pi-function](pi_function.png)
+
+#### Gaussian function
+
```

Comment:
Since you have included screenshots of the graphs for all the other functions, it would only be consistent to include a graph for a Gaussian function too